src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -118,6 +118,11 @@ class WorkspaceSettingsPage extends React.Component {
             return <FullScreenLoadingIndicator />;
         }
 
+        // Disables button if none of the form values have changed or workspace name empty
+        const isButtonDisabled = (this.state.name.trim() === ''
+            || (this.props.policy.name === this.state.name.trim()
+            && this.props.policy.outputCurrency === this.state.currency));
```

Comment:
```suggestion
        const isButtonDisabled = this.state.name.trim() === ''
            || (this.props.policy.name === this.state.name.trim()
            && this.props.policy.outputCurrency === this.state.currency);
```

Reply:
Thanks for the correction. Pushed commit.